sails.io.js

```diff
@@ -939,6 +945,28 @@
 
         self.on('disconnect', function() {
           self.connectionLostTimestamp = (new Date()).getTime();
+          // If there is a list of registered callbacks, call each of them with an error
+          // and then wipe the list.
+          if (typeof self.responseCbs === 'object' && self.responseCbs.length) {
```

Comment:
from what I can see here, I don't think there's any way responseCbs could be the `null` literal, so we should be safe without an additional truthiness check here (`&& self.responseCbs`).  Even still, might be worth it to be explicit to defend against short-sightedness of our future selves (or I guess in the extremely weird edge case where someone sets this to `null` manually in userland on their one-off SailsSocket instance or on `io.socket`)

Reply:
whoops nevermind (see subsequent changes- this becomes irrelevant)